L/libzenohc/build_tarballs.jl

```diff
@@ -3,29 +3,32 @@
 using BinaryBuilder, Pkg
 
 name = "libzenohc"
-version = v"0.10.1"
+version = v"1.0.0"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/eclipse-zenoh/zenoh-c.git", "a0bacbb65b11e1e5ecedb3b3888beba78bf227cd")
+    GitSource("https://github.com/eclipse-zenoh/zenoh-c.git", "bca57bb6e5a55421cc6ef20ddae5fd3cbb58c30c")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
+cd $WORKSPACE/srcdir
 cd $WORKSPACE/srcdir/zenoh-c
-cmake -B build -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DZENOHC_CARGO_CHANNEL=$RUSTUP_TOOLCHAIN -DZENOHC_CUSTOM_TARGET=$CARGO_BUILD_TARGET
+cmake -B build -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DZENOHC_CUSTOM_TARGET=$CARGO_BUILD_TARGET
 cmake --build build --parallel ${nproc}
-cmake --install build
+cmake --build build --target install
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = [
+    Platform("aarch64", "macos"; ),
```

Comment:
This changed the order of the platforms, which (a) makes review harder than necessary and (b) breaks git blame.